src/components/bulk-email-tool/bulk-email-form/bulk-email-recipient/components/EmailList/utils.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable import/prefer-default-export */
+export const isValidEmail = (email) => {
+  const emailRegex = /^(([^<>()·=~ºªÇ¨?¿*^|#¢∞¬÷"$%"≠´}{![\]\\.,;:\s@"]+(\.[^<>·$%&/=~ºªÇ¨?¿*^|#¢∞¬÷""≠´}{!()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
Is this the same validation that the https://github.com/openedx/frontend-app-authn runs when registering a user (without custom registration rules)?

Reply:
They have a regex for that as well [here](https://github.com/openedx/frontend-app-authn/blob/332d6abee748b3e74f3d20d0ef334fac2f9a52d8/src/data/constants.js#L31)

Reply:
can we use that one? So we know it's maintained elsewhere. We can add a comment where we got it from